src/rabbit_control_main.erl

```diff
@@ -679,6 +693,29 @@ default_if_empty(List, Default) when is_list(List) ->
        true       -> [list_to_atom(X) || X <- List]
     end.
 
+wait_info_messages(Pid, Ref, ArgAtoms, Timeout) ->
```

Comment:
Okay noted, I'll update & bear that in mind. And yes, purpose is to wait for info massages.